baseapp-core/baseapp_core/tests/fixtures.py

```diff
@@ -9,9 +9,9 @@
 from asgiref.sync import sync_to_async
 from rest_framework.test import APIClient
 
-from ..graphql.testing import graphql_query
+import testproject.tests.factories as f
```

Comment:
I think this can break for projects importing fixtures from this file, I think it shouldn't import factories from the testproject here